app/controllers/dfe_sign_in_controller.rb

```diff
@@ -1,9 +1,13 @@
 class DfESignInController < ActionController::Base
   protect_from_forgery except: :bypass_callback
+  include SyncWithDsiProfile
 
   def callback
     DfESignInUser.begin_session!(session, request.env['omniauth.auth'])
 
+    user = get_support_user || ProviderUser.load_from_session(session)
+    update_profile_from_dfe_sign_in(dfe_user: DfESignInUser.load_from_session(session), local_user: user) if user
```

Comment:
Perhaps use this directly, so we don't have to `include` the module.

```suggestion
    SyncWithDsiProfile.update_profile_from_dfe_sign_in(dfe_user: DfESignInUser.load_from_session(session), local_user: user) if user
```

Including modules / using concerns tends to become tricky if an app grows because it's hard to ensure there aren't clashes in method names (private methods are also included), and it's not easy to find out where a method is defined.

Reply:
Ok